lib/src/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+if(NOT DEFINED ENV{FLUTTER_HOME})
+  message(FATAL_ERROR "Flutter home must be defined")
+endif()
+
+add_library(
+  bindings SHARED ${DART_WORMHOLE_WILLIAM_C_DIR}/async_callback.c
+                  $ENV{FLUTTER_HOME}/bin/cache/dart-sdk/include/dart_api_dl.c)
+target_include_directories(
+  bindings PRIVATE $ENV{FLUTTER_HOME}/bin/cache/dart-sdk/include)
+target_include_directories(bindings PRIVATE ${flutter_INCLUDE_DIRECTORIES})
```

Comment:
Double-check if needed or redundant.

Reply:
Removed. The preceding include directory is sufficient.